### PR TITLE
Key Vault: refactored kwarg docstrings for certificates

### PR DESCRIPTION
**Pull request by iscai-msft**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py [hunk 1]
```diff
@@ -289,17 +287,16 @@ class CertificateClient(AsyncKeyVaultClientBase):
         :param str name: The name of the certificate.
         :param bytes certificate_bytes: Bytes of the certificate object to import.
             This certificate needs to contain the private key.
+        :keyword bool enabled: Whether the certificate is enabled for use.
+        :keyword tags: Application specific metadata in the form of key-value pairs.
+        :paramtype tags: dict[str, str]
+        :keyword str password: If the private key in the passed in certificate is encrypted, it
+         is the password used for encryption.
+        :keyword policy: THe management policy for the certificate
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py]: T**h**e

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py [hunk 2]
```diff
@@ -784,18 +775,17 @@ def list_certificate_versions(self, name: str, **kwargs: "**Any") -> AsyncIterab
 
         :param str name: The name of the issuer.
         :param str provider: The issuer provider.
+        :keyword bool enabled: Whether the issuer is enabled for use.
+        :keyword str account_id: The user name/account name/account id.
+        :keyword str password: The password/secret/account key.
+        :keyword str organization_id: Id of the organization
+        :keyword admin_details: Details of the organization administrators of the
+         certificate issuer.
+        :keyword admin_details: list[~azure.keyvault.certificates.models.AdministratorDetails]
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py]: `:paramtype admin_details:`?

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py [hunk 3]
```diff
@@ -34,16 +34,14 @@
 
 
 class CertificateClient(KeyVaultClientBase):
-    """A high-level interface for managing a vault's certificates.
+    """A high-level interface for managing a vault's keys.
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py]: :suspect:

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py [hunk 4]
```diff
@@ -305,17 +306,16 @@ def import_certificate(self, name, certificate_bytes, **kwargs):
         :param str name: The name of the certificate.
         :param bytes certificate_bytes: Bytes of the certificate object to import. This certificate
             needs to contain the private key.
+        :keyword bool enabled: Whether the certificate is enabled for use.
+        :keyword tags: Application specific metadata in the form of key-value pairs.
+        :paramtype tags: dict[str, str]
+        :keyword str password: If the private key in the passed in certificate is encrypted, it
+         is the password used for encryption.
+        :keyword policy: THe management policy for the certificate
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py]: T**h**e

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py (hunk 1 above)
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py]: thanks!

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py (hunk 3 above)
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py]: oh no i'm so sorry

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py (hunk 4 above)
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py]: changed, thanks!

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py (hunk 2 above)
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py]: switched, tahnks!

**Pull request merged**